src/foam/nanos/auth/login/SignIn.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
Tried running this locally, the LoginView seems really narrow?

<img width="528" alt="Screenshot 2024-11-23 at 12 21 38 PM" src="https://github.com/user-attachments/assets/5373c343-df03-42ad-af56-feb7767defcf">

Reply:
I've never seen this particular view/layout. 
What are your steps to reproduce?

Reply:
looks like https://github.com/kgrgreer/foam3/pull/3585 re-does this view significantly, this is probably an irrelevant comment then

Reply:
Addressed.